Arduino/McLighting/request_handlers.h

```diff
@@ -1497,7 +1497,7 @@ bool readStripConfigFS(void) {
         DBG_OUTPUT_PORT.println(" Parsed!");
         JsonObject json = jsonBuffer.as<JsonObject>();
         serializeJson(json, DBG_OUTPUT_PORT);
-        WS2812FXStripSettings.stripSize = json["pixel_pount"];
+        WS2812FXStripSettings.stripSize = (json["pixel_count"]) ? json["pixel_count"] : NUMLEDS;
         WS2812FXStripSettings.RGBOrder = json["rgb_order"];
```

Comment:
Can we parse all `json["xxxxx"]` to `json["xxxxx"].as<int>()`

Reply:
I made changes to how json is parsed out. Hope I didn't break anything.

Reply:
I will test this evening if I get a chance. On top of that, now that I have it running again, it doesn’t seem to be saving the changes (speed and brightness specifically) across restarts. I will have to look further into this when I have time.

Reply:
That could be because SPIFFs in not enabled (1M Program/0M SPIFFs) in esp9266-core v2.5.0

Reply:
I have the D1 Mini selected, and flashing as 3MB Program / 1MB SPIFFS. It shows up, and can confirm that it previously held files and settings prior to the breakage. To my knowledge, I have not updated my esp8266 library since, unless it is updated automatically.